addon/components/o-s-s/side-panel/header.hbs

```diff
@@ -0,0 +1,7 @@
+<div class="oss-side-panel__header__content fx-row fx-xalign-center">
+  {{#if @onReturn}}
+    <OSS::Icon @icon="fa-solid fa-arrow-left" role="button" class="margin-right-px-12" {{on "click" @onReturn}} />
+  {{/if}}
+  <h4 class="fx-1 margin-px-0">{{@title}}</h4>
+  <OSS::Button @size="sm" @icon="fa-xmark" @square={{true}} {{on "click" @onClose}} />
```

Comment:
`fa-times` is what we've been using in other components. What are the differences ?